var/spack/repos/builtin/packages/netlib-lapack/package.py

```diff
@@ -49,6 +49,8 @@ class NetlibLapack(CMakePackage):
     patch('ibm-xl-3.9.1.patch', when='@3.9.1 %xl')
     patch('ibm-xl-3.9.1.patch', when='@3.9.1 %xl_r')
 
+    patch('fj.patch', when='@3.7: %fj')
```

Comment:
Can you add a comment above the directive explaining briefly what the patch does and where does it come from?

Reply:
@h-murai